pkg/k8s/apps/translate.go

```diff
@@ -138,6 +138,8 @@ func translate(t *model.Translation, c *kubernetes.Clientset, isOktetoNamespace
 		TranslatePodSecurityContext(&t.K8sObject.GetPodTemplate().Spec, rule.SecurityContext)
 		TranslatePodServiceAccount(&t.K8sObject.GetPodTemplate().Spec, rule.ServiceAccount)
 		TranslateOktetoDevSecret(&t.K8sObject.GetPodTemplate().Spec, t.Name, rule.Secrets)
+		TranslateOktetoNodeSelector(&t.K8sObject.GetPodTemplate().Spec, rule.NodeSelector)
```

Comment:
Could you call this two from `TranslateDevContainer`? it makes things easier when moving this code to the okteto backend mutation webhook